lib/stringtheory/src/lib.rs

```diff
@@ -111,17 +97,17 @@ impl MetaString {
     /// This does not allocate.
     pub const fn empty() -> Self {
         Self {
-            inner: Inner::Shared(bytes::Bytes::new()),
+            inner: Inner::Owned(String::new()),
```

Comment:
For the sake of reviewers: the change here is a no-op because `String` wraps over `Vec<u8>`, which in fact does not allocate when created via `Vec::new`, and only allocates when first trying to push something into the vector.